test/Test/HTTPure/TestHelpers.js

```diff
@@ -1,16 +1,16 @@
-"use strict";
-
-exports.mockRequestImpl = function(httpVersion) {
-  return function(method) {
-    return function(url) {
-      return function(body) {
-        return function(headers) {
-          return function() {
-            var stream = new require('stream').Readable({
-              read: function(size) {
+import * as zstream from "stream";
+
+export function mockRequestImpl(httpVersion) {
+  return function (method) {
+    return function (url) {
+      return function (body) {
+        return function (headers) {
+          return function () {
```

Comment:
Do we use arrow functions in the new hotness?  This would be way cleaner IMHO as

```js
export const mockRequestImpl = httpVersion => method => url => body => headers => () => {
    ...
};
```

Reply:
Yep! Arrow functions are fair game.

Reply:
Sweet, I'm going to make this change then.  Thanks.